lms/static/scripts/frontend_apps/components/LMSGrader.js

```diff
@@ -19,30 +18,34 @@ export default function LMSGrader({
   assignmentName,
   courseName,
   students,
-  onChangeSelectedUser,
 }) {
   // No initial current student selected
   const [currentStudentIndex, setCurrentStudentIndex] = useState(-1);
 
+  /**
+   * Makes an RPC call to the sidebar to change to the focused user.
+   *
+   * @param {User} user - The user to focus on in the sidebar
+   */
+  const changeFocusedUser = async user => {
```

Comment:
This is much cleaned up now, all the logic for setting the focus to the sidebar is done here. We no longer need to tamper with the config in the DOM